### PR TITLE
Glennc/progressbar

### PR DESCRIPTION
**Pull request by glennc**
Adds progress bars when downloading.

### CODE REVIEW

**Comment by msotclas** [comment]
Hi __@glennc__, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <p>
        It looks like you're a Microsoft contributor (Glenn Condron). If you're full-time, we DON'T require a Contribution License Agreement. If you are a vendor, please DO sign the electronic Contribution License Agreement. It will take 2 minutes and there's no faxing! https://cla2.msopentech.com.
    </p>

TTYL, MSOTBOT;

File: src/dnvm.ps1 [hunk 1]
```diff
@@ -927,6 +959,7 @@ function dnvm-install {
     }
 
     if ($VersionNuPkgOrAlias -eq "latest") {
+		Write-Progress -Activity "Installing runtime" "Determining latest runtime" -Id 1
```
Comment (Eilon) [inline src/dnvm.ps1]: \t\t\t\t :frowning: :panda_face:

File: src/dnvm.ps1 [hunk 2]
```diff
@@ -997,6 +1034,7 @@ function dnvm-install {
             if (-not $NoNative) {
                 if ((Is-Elevated) -or $Ngen) {
                     $runtimeBin = Get-RuntimePath $runtimeFullName
+                    Write-Progress -Activity "Installing runtime" "Generating runtime native images for your machine" -Id 1
```
Comment (Eilon) [inline src/dnvm.ps1]: Recommend omitting the "for your machine" part (from all the messages)

**Comment by anurse** [comment]
:shipit:

**Pull request merged**